Ginger/Ginger/ALM/ALMConnectionPage.xaml.cs

```diff
@@ -344,16 +354,14 @@ public void ShowAsWindow()
 
         private void CloseWindow(object sender, EventArgs e)
         {
-            ALMIntegration.Instance.SyncConfigurations();
             _pageGenericWin.Close();
         }
 
         private void StyleRadioButtons()
         {
-            switch ((ALMIntegration.eALMType) WorkSpace.Instance.Solution.AlmType)
+            switch (CurrentAlmConfigurations.AlmType)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)